docs/sources/alerting/notifications.md

```diff
@@ -230,3 +230,14 @@ Notification services which need public image access are marked as 'external onl
 
 All alert notifications contain a link back to the triggered alert in the Grafana instance.
 This URL is based on the [domain]({{< relref "../administration/configuration/#domain" >}}) setting in Grafana.
+
+## Templating
+
+> **Note:** Alert notification templating is only available in Grafana v7.4 and above.
+
+You can apply data templating to an alert notification's title and message by referring to the matching alert condition query's [labels]({{< relref "../getting-started/timeseries-dimensions.md#labels" >}}). Template variables are formatted as follows: `${Label}`.
```

Comment:
Referring to "Template variables" here can be confusing and mixed up with dashboard template variables.

Nit. A note about how multiple labels are handled would be nice.